### PR TITLE
improvement: ARSN-25 update version to 8.1.5

### PR DESCRIPTION
**Pull request by naren-rajendran**
Update version to 8.1.5

### CODE REVIEW

**Comment by naren-rajendran** [comment]
@bert-e reset

**Review by miniscruff — APPROVED** [review]
I think we should update the package json and then create a git tag that way if you npm install it has the same version.

**Comment by naren-rajendran** [comment]
> I think we should update the package json and then create a git tag that way if you npm install it has the same version.

Yes, I missed it.

**Review by nicolas2bert — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=8"
   },
-  "version": "8.1.3",
+  "version": "8.1.5",
```
Comment (jonathan-gramain) [inline package.json:6]: Should it be bumped to 8.1.6, since the tag 8.1.5 already exists?

Comment (naren-rajendran) [reply package.json:6]: I removed the one created earlier.

**Pull request merged**